project/src/test/java/com/google/sps/data/PlacesTest.java

```diff
@@ -89,6 +89,29 @@ public void filter_tooLowRating_filterOut() {
     assertEquals(result, ImmutableList.of(highRatingPlace));
   }
 
+  @Test
+  public void filter_floatingPointRating_filterOutByRoundingToNearestInt() {
+    // This test checks that the rating is filtered after rounding it to the nearest int.
+    // For example, when places with 5 starts are desired, places that have at least 4.5 stars will
```

Comment:
There is a little issue with rint (that I am fine with, just pointing it out) that in case of 0.5 at decimal places it will round off to nearest even double value. This means that 4.5 will round to 4 and will not be supplied if 5 was the threshold.

Reply:
Yes you are right, I also noticed it. I changed the documentation to make it clearer. Other than that do you think something should be done?

Reply:
No I think its fine, just the documentation didn't exactly fit but now it's good :)